### PR TITLE
Update to publish binaries

### PR DESCRIPTION
**Pull request by cloudnautique**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -7,3 +7,159 @@ pipeline:
...
+      branch: [master, v1.6]
+      event: tag
+      ref:
+        exclude: [ refs/tags/*rc* ]
+
+  github-binary-publish-prerelease:
+    image: plugins/github-release
+    prerelease: true
+    files:
+     - dist/artifacts/latest/*
```
Comment (deniseschannon) [inline .drone.yml]: Shouldn't this be? `source: dist/artifacts/v*`

File: .drone.yml [hunk 2]
```diff
@@ -7,3 +7,159 @@ pipeline:
...
+    checksum:
+     - sha256
+    secrets: [github_token]
+    when:
+      branch: [master, v1.6]
+      event: tag
+      ref:
```
Comment (deniseschannon) [inline .drone.yml]: needs to be removed

File: .drone.yml [hunk 3]
```diff
@@ -7,3 +7,159 @@ pipeline:
...
+    source: dist/artifacts/v*
+    target: releases.rancher.com/cli
+    acl:
+      - allUsers:READER
+    cache_control: public,max-age=3600
+    secrets:
+      - source: google_auth_key
+        target: GOOGLE_CREDENTIALS
+    when:
+      branch: master
```
[the remark this answers is not in the document]
Comment (cloudnautique) [reply .drone.yml]: master is correct for v2

**Comment by deniseschannon** [comment]
LGTM

**Pull request merged**